examples/js/effects/VREffect.js

```diff
@@ -134,7 +134,7 @@ THREE.VREffect = function ( renderer, onError ) {
 
 			}
 
-			renderer.setPixelRatio( 1 );
+			renderer.setPixelRatio( rendererPixelRatio );
 			renderer.setSize( eyeWidth * 2, eyeHeight, false );
```

Comment:
I believe that when rendered in a headset you will be applying the pixelRatio of the main display instead of the headset one.